x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/gcp_credential_form.tsx

```diff
@@ -71,7 +74,21 @@ const GCPSetupInfoContent = () => (
   </>
 );
 
-const GoogleCloudShellSetup = () => {
+const GoogleCloudShellSetup = ({
+  fields,
+  onChange,
+  input,
+}: {
+  fields: Array<GcpFields[keyof GcpFields] & { value: string; id: string }>;
+  onChange: (key: string, value: string) => void;
+  input: NewPackagePolicyInput;
+}) => {
+  const accountType = input.streams?.[0]?.vars?.['gcp.account_type']?.value;
```

Comment:
there's an error related to the account type being required whenever the page loads, however, this looks more like an integration error.

![image](https://github.com/elastic/kibana/assets/19270322/b6b173ca-e4f6-4610-a98c-2d970deb527b)

Reply:
Looks like it is a known issue: https://github.com/elastic/security-team/issues/7686

Reply:
@opauloh we couldn't reproduce it, could it be a race condition ? are you able to reproduce it?